e2e_samples/dataset_versioning/datafactory/config/README.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+This folder is for SQL query which creata following required tables and stored procedures.
+
+- `source` table which stores sample data
+- `WaterMark` table storing table name and its watermark value
+- `usp_UpdateWatermark` stored procedure to update watermark value
+
+## How to create required tables and stored procedure
+
+1. Install [Azure Data Studio](https://docs.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver15)
+1. Get user id and connection string for SQL Database
+   1. Sign-in to Azure and go to provisioned resource group.
+   1. Go to Azure KeyVault
+   1. Click [Access policies] in left pane, then click [+ Add Access Policy]
```

Comment:
I suggest describing what you are doing in these steps since there are a lot of steps.

Also (related to permissions)  I did not need to add an access policy and it still worked for me

Reply:
Have updated. See detail in https://github.com/Azure-Samples/modern-data-warehouse-dataops/pull/305